.github/workflows/lint-ts.yml

```
@@ -43,3 +43,14 @@ jobs:
                   npm install @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint-plugin-tsdoc eslint typescript eslint-plugin-import@latest eslint-config-prettier
                   npm i
                   npx eslint .
+
+            - name: Iterate over folders and prettier
+                run: |
```

```suggestion
              run: |
```
We need a repo-wide linter for yml 😉